treeherder/perf/models.py

```diff
@@ -270,6 +270,7 @@ class PerformanceAlertSummary(models.Model):
     )
 
     created = models.DateTimeField(auto_now_add=True, db_index=True)
+    triage_due = models.DateTimeField(null=True, default=None)
```

Comment:
Instead of `triage_due` what do you think about `triage_date` or `triage_due_date`